apps/ledger-live-desktop/src/renderer/components/OperationsList/index.tsx

```diff
@@ -114,6 +115,21 @@ export class OperationsList extends PureComponent<Props, State> {
         })
       : undefined;
 
+    // -- THIS CAN BE REMOVED ONCE THE DATE ERROR HAS BEEN FIGURED OUT
+    if (groupedOperations?.sections) {
+      for (const group of groupedOperations.sections) {
+        const { day } = group;
+        if (day instanceof Date && !isNaN(day as unknown as number)) {
+          log("Ethereum Date Error", "Date in operation is invalid", {
+            day,
+            accountId: account?.id,
+            groupOps: group.data,
+          });
+        }
+      }
+    }
+    // -- THIS CAN BE REMOVED ONCE THE DATE ERROR HAS BEEN FIGURED OUT
```

Comment:
i think it is preferable to do this in the logic side. (groupAccountsOperationsByDay function) so you don't have to do a loop again but just contain the temporary code inside the place it's generating the groups.

btw the date comes from `bestOp.date` to it means one of the coin impl produces Date that are NaN. 

I have seen this issue in the past (on sentry i think) and this is happening whenever `new Date(undefined)` happens, so indeed there is lot of place to this occurence, but sounds like a backend not returning a date where we expected it. it sounds a good idea to ignore the operation in that case because it can produces pretty ugly error on the end UI as the date is today assumed to always be settled.